src/docs/content/reference/current/http/websocket/index.md

```diff
@@ -123,7 +123,7 @@ WebSocket support introduces new HttpProtocol parameters:
 You can inspect WebSocket traffic if you add the following logger to your logback configuration:
 
 ```xml
-<logger name="io.gatling.http.action.ws.fsm" level="DEBUG" />
+<logger name="io.gatling.http.action.ws.WsLogger" level="DEBUG" />
```

Comment:
We're losing the other logs then. Also, it's a change users might not notice when upgrading. We could move WsLogger in fsm instead.